spec/backend/utils/base-decorator.spec.js

```diff
@@ -30,7 +19,7 @@ describe('BaseDecorator', function () {
         }
         overwritenFunction() { return 'overwritenFunctionValue' }
       }
-      this.decorator = new Decorator(this.mockedResource)
+      this.decorator = new Decorator(this.args)
```

Comment:
I changed entire flow in a way that now Decorator accepts in constructor adminBro object => we have access to helpers in decorator by default, we dont have to pass them from the view